extensions/wikia/AffiliateService/js/ext.AffiliateService.js

```diff
@@ -0,0 +1,100 @@
+// find the first slot where we can insert
+// todo: we could use a better algorithm here
+function insertSpot(arr, val) {
+	for (var i = 0; i < arr.length - 1; i++) {
+		var arrVal = arr[i];
+		var nextVal = arr[i + 1];
+
+		if (i === 0 && arrVal > val) {
+			return -1;
+		}
+
+		if (arrVal <= val && nextVal > val) {
+			return i;
+		}
+	}
+
+	return arr.length;
+}
+
+require(['jquery', 'mw'], function ($, mw) {
+	'use strict';
+	$(function () {
+		var $infoBox = $('.portable-infobox').first();
+
+		if (!$infoBox) {
```

Comment:
I'm surprised this works. IIRC it would return a `jQuery` object regardless of what it contains. I'd think you want to check its `.length`.